packages/office-ui-fabric-react/src/components/Tooltip/TooltipHost.tsx

```diff
@@ -82,10 +82,11 @@ export class TooltipHost extends BaseComponent<ITooltipHostProps, ITooltipHostSt
             targetElement={ this._getTargetElement() }
             directionalHint={ directionalHint }
             directionalHintForRTL={ directionalHintForRTL }
-            calloutProps={ assign(calloutProps, {
-              onMouseEnter: this._onTooltipMouseEnter,
-              onMouseLeave: this._onTooltipMouseLeave
-            }) }
+            calloutProps={ assign({},
```

Comment:
much better..... the other option is to use spreads (that seems a bit more in vogue these days)